frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-hub.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { checkErrors, testName } from '../../../integration-tests-cypress/support';
+import { nav } from '../../../integration-tests-cypress/views/nav';
+
+describe('Interacting with OperatorHub', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it('displays OperatorHub tile view with expected available Operators', () => {
+    cy.log('navigate to OperatorHub');
+    nav.sidenav.clickNavLink(['Operators', 'OperatorHub']);
+    cy.url().should('include', `/operatorhub/ns/${testName}`);
+    cy.log('more than one tile should be present');
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+  });
+
+  it('displays OperatorHub tiles filtered by "Provider Type"', () => {
+    cy.log('enable the Red Hat filter');
+    cy.byTestID('providerType-red-hat').click();
+    cy.log('more than one tile should be present');
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+    cy.log('track which tile is first');
+    const firstOperatorWithRHFilterApplied = cy
```

Comment:
I'm not sure this is working the way you expect.  Pls see: https://docs.cypress.io/guides/references/best-practices.html#Assigning-Return-Values.   You'll probably want to use closures as descripted [here](https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Return-Values).

Reply:
I think something like:
```
cy.get('.catalog-tile-pf').first().find('.catalog-tile-pf-title').then(($catalogTitle => {
    cy.log('diable the Red Hat filter');
    cy.byTestID('providerType-red-hat').click();
    cy.log('enable the Certified filter');
    cy.byTestID('providerType-certified').click();
    cy.log('more than one tile should be present');
    cy.get('.catalog-tile-pf')
      .its('length')
      .should('be.gt', 0);
    cy.log('the first tile for Certified should not be the same as Red Hat');
    cy.get('.catalog-tile-pf')
      .first()
      .find('.catalog-tile-pf-title')
      .should('not.equal', catalogTitle);  // maybe catalogTitle.text or something
})
```